javascript.html.markdown

```diff
@@ -201,7 +220,34 @@ myObj.myThirdKey = true;
 myObj.myFourthKey; // = undefined
 
 ///////////////////////////////////
-// 3. Logic and Control Structures
+// 3. Modules, Imports and Exports
+
+// In ES6 environments, you can specify variables to export when your script is
+// imported from another script.
+export var myNumber = 42;
+export var otherNumber = 63;
+
+// You can also write the above as follows:
+var myNumber = 42;
+export myNumber;
+
+// To import variables from another script, use `import`.
+import * as numbers from "numbers"; // file imported is numbers.js
+numbers.myNumber === 42; // true
+
+// You can also specific variables/functions.
+import myNumber from "numbers";
+myNumber === 42; // true
+
+// When importing multiple specific variables, surround them with curly braces.
```

Comment:
You always need curly braces for the kind of import introduced here. You're not mentioning default exports, which is what gets imported without the curly braces.